docs/navigation-configuration.md

```diff
@@ -58,6 +59,10 @@ window.Luigi.setConfig({
 
 ### Navigation parameters
 
+- **nodeAccessibilityResolver** allows you to define a permission checker function that gets executed on every Node. If it returns false, the Node and it's children are removed from the navigation structure. You may want to add custom values to Nodes to receive it in the **nodeAccessibilityResolver** function. See [angular sampleconfig.js](../core/examples/luigi-sample-angular/src/assets/sampleconfig.js) for our constraints example.
+
+## Nodes
```

Comment:
Is `##` instead of`###`intentional here?

Reply:
Yes, since this is a standalone feature while Nodes section is a collection of settings for a Node.

    {
      navigation: {
        nodeAccessibilityResolver: someFunction,
        nodes: [node1, node2] // every nodeN consists of parts described in ## Nodes
      }
    }